lib/rmg/threadpool/MapProxy.h

```diff
@@ -28,7 +28,7 @@ class MapProxy
     void insertObjects(std::set<CGObjectInstance*>& objects);
     void removeObject(CGObjectInstance* obj);
 
-    void drawTerrain(CRandomGenerator & generator, std::vector<int3> & tiles, TerrainId terrain);
+	void drawTerrain(CRandomGenerator & generator, std::vector<int3> & tiles, TerrainId terrain);
```

Comment:
Unneccessary tab

Reply:
Actually looks like entire file uses spaces instead of tabs (except for 1 line changed by me)